From 759b1ed4c7a3a8a5621cc12d9cff7334f0e3e957 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Mon, 26 Nov 2018 20:15:16 +0000
Subject: [PATCH 214/432] overlays: Update upstream overlay

The vc4-kms-v3d overlay gained an extra fragment enabling the txp node,
so rebuild the upstream overlay to match.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/overlays/upstream-overlay.dts | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/upstream-overlay.dts b/arch/arm/boot/dts/overlays/upstream-overlay.dts
index ff37a5013de4..b597f863c4a8 100644
--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -116,6 +116,12 @@
 		};
 	};
 	fragment@17 {
+		target = <&txp>;
+		__overlay__ {
+			status = "okay";
+		};
+	};
+	fragment@18 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
@@ -130,21 +136,21 @@
 			status = "okay";
 		};
 	};
-	fragment@18 {
+	fragment@19 {
 		target = <&uart1>;
 		__overlay__ {
 			interrupt-parent = <&intc>;
 			interrupts = <0x1 0x1d>;
 		};
 	};
-	fragment@19 {
+	fragment@20 {
 		target = <&spi1>;
 		__overlay__ {
 			interrupt-parent = <&intc>;
 			interrupts = <0x1 0x1d>;
 		};
 	};
-	fragment@20 {
+	fragment@21 {
 		target = <&spi2>;
 		__overlay__ {
 			interrupt-parent = <&intc>;
-- 
2.19.1

